west.yml

```diff
@@ -198,7 +198,7 @@ manifest:
       groups:
         - hal
     - name: hal_nxp
-      revision: 272f84ca60afc759d81ed94100e78b37a859d3ff
+      revision: pull/391/head
```

Comment:
Update this to point to HAL PR 415

Reply:
Yes, I know. I have modified to 415, however, some cases build failed with 415PR. https://github.com/zephyrproject-rtos/zephyr/actions/runs/10195617084/job/28206275931
I want to get a successful result based on 391 firstly, then I will change revision to 415 tomorrow.

Reply:
Updated to PR 415.